ci/before_install.sh

```diff
@@ -40,7 +40,8 @@ fi
 
 if [[ "${TRAVIS_OS_NAME}" == osx ]]; then
   echo "Update default Ruby"
-  brew switch ruby 2.2.5
+  brew install ruby
+  brew switch ruby
```

Comment:
This failed because of missing version:

    Error: Usage: brew switch <name> <version>

If brew wasn't used to install ruby (depends on travis default env) I think merely upgrading it won't work.